src/lucky_web/page_helpers/number_to_currency.cr

```diff
@@ -0,0 +1,33 @@
+module LuckyWeb::NumberToCurrency
+  DEFAULT_PRECISION       = 2
+  DEFAULT_UNIT            = "$"
+  DEFAULT_SEPARATOR       = "."
+  DEFAULT_DELIMITER       = ","
+  DEFAULT_DELIMITER_REGEX = /(\d)(?=(\d\d\d)+(?!\d))/
+  DEFAULT_FORMAT          = "%u%n"
+
+  def number_to_currency(value : Nil, **options)
+  end
+
+  def number_to_currency(value : Float | Int32, precision : Int32 = DEFAULT_PRECISION, unit : String = DEFAULT_UNIT, separator : String = DEFAULT_SEPARATOR, delimiter : String = DEFAULT_DELIMITER, delimiter_pattern : Regex = DEFAULT_DELIMITER_REGEX, format : String = DEFAULT_FORMAT, negative_format : String = DEFAULT_FORMAT)
+    number_to_currency(value.to_s, precision: precision, unit: unit, separator: separator, delimiter: delimiter, delimiter_pattern: delimiter_pattern)
+  end
+
+  def number_to_currency(value : String, precision : Int32 = DEFAULT_PRECISION, unit : String = DEFAULT_UNIT, separator : String = DEFAULT_SEPARATOR, delimiter : String = DEFAULT_DELIMITER, delimiter_pattern : Regex = DEFAULT_DELIMITER_REGEX, format : String = DEFAULT_FORMAT, negative_format : String = DEFAULT_FORMAT)
```

Comment:
Rather than having two methods, you could do this:

```crystal
def number_to_currency(value : String | Int32 | Float, ...)
  value = value.to_s # Force it into a string. This way you don't need two different methods
```